packages/map/src/overlay/markers/poi-dot-marker.tsx

```diff
@@ -8,36 +8,45 @@ import React, {
 } from 'react'
 import { OverlayView, OverlayViewProps } from '@react-google-maps/api'
 
-import { BubbleMarker, DotMarker } from './primary-marker'
 import {
+  BubbleMarker,
   CIRCLE_MARKER,
   CircleType,
-  MarkerBaseProps,
-} from './primary-marker/circle-marker/circle-marker-base'
+  DotMarker,
+} from './primary-marker'
+import { MarkerBaseProps } from './primary-marker/circle-marker/circle-marker-base'
 
 export interface DotWithPopOverMarkerProps
   extends Pick<MarkerBaseProps, 'zIndex'>,
     Omit<OverlayViewProps, 'mapPaneName'> {
   type: CircleType
   active: boolean
-  bubbleContent: React.ComponentType
-  activeWithDot?: boolean
+  dotSize?: { width: number; height: number }
+  bubbleContent: React.ReactNode
+  activeContent?: React.ReactNode
+  inActiveContent?: React.ReactNode
```

Comment:
`inactive`는 한 단어입니당. 그런데 `activeWithDot`이란 prop이 사라져버려서.. Breaking change로 봐야 하나요 요거?

```suggestion
  inactiveContent?: React.ReactNode
```

Reply:
content-web, trips-web , hotels-web에는 해당사항이 없어서 6.1.1을 적용하면 되는데
genie-web에는 해당사항이 있어 patch 버전업하고 적용하려고 합니다!

Reply:
ㅋㅋㅋ 아직 사용처가 없으니까 익스큐즈하는 거군요